src/app/routes/connector-ui/connector-ui-routing.module.ts

```diff
@@ -61,6 +62,7 @@ export const routes: Routes = [
       requiresFeature: 'logout-button' as EdcUiFeature,
     },
   },
+  {path: '**', component: PageNotFoundComponent, data: {hideInNav: true}},
```

Comment:
Architecture: Did it not work to add the route to the AppRoutingModule?

This would allow you to:
- Only have one route (for both route sets)
- Have the component in `routes/page-not-found` (I think that's ok)
- Declare the component in `app-module.ts`
- Not have to create a common "ErrorPagesModule", which you would have to do if you wanted to put components into the component library.